astropy/tests/tests/test_runner.py

```diff
@@ -1,9 +1,11 @@
-from astropy.tests.runner import TestRunner, TestRunnerBase, keyword
+from astropy.tests.runner import TestRunner as T_Runner
```

Comment:
@mhvk had the same objection! :ghost: 

Let's see if mhvk/astropy#23 gets rid of the remaining warnings. If that is merged, this PR will automatically update with those changes.